pkgs/development/tools/xcbuild/platform.nix

```diff
@@ -0,0 +1,604 @@
+{ stdenv, sdk, writeText, platformName, xcbuild }:
+
+let
+
+  Info = {
+    CFBundleIdentifier = platformName;
+    Type = "Platform";
+  };
+
+  Version = {
+    ProjectName = "OSXPlatformSupport";
+  };
+
+  PackageTypes = [
```

Comment:
@grp Should these rules be added to xcbuild? It seems like every xcode project will need some of these.